src/reviews/reviews.controller.ts

```diff
@@ -42,6 +43,7 @@ export class ReviewsController {
 
   @ApiBearerAuth()
   @Post()
+  @Throttle({ default: { limit: 3, ttl: 60000 } })
```

Comment:
Move limit and ttl value to env so adjust them without making changes to code later.